src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -1435,6 +1435,26 @@ else if (source == speechActivity)
         {
             speechActivityPropertyChange(ev);
         }
+        else if (Endpoint.SELECTED_ENDPOINTS_PROPERTY_NAME.equals(ev.getPropertyName()))
+        {
+            Set<String> oldSelectedEndpoints = (Set<String>)ev.getOldValue();
+            Set<String> newSelectedEndpoints = (Set<String>)ev.getNewValue();
+            // Any endpoints in the oldSelectedEndpoints list which AREN'T
+            // in the newSelectedEndpoints list should have their count decremented
+            oldSelectedEndpoints.stream()
+                .filter(oldSelectedEp -> !newSelectedEndpoints.contains(oldSelectedEp))
+                .map(this::getEndpoint)
```

Comment:
I wonder if calling `getEndpoint` (which has to synchronize and clean the list) each time is ideal, but I guess it's not a problem since the lists should have a single element most of the time.